apps/platform/src/app/(main)/page.tsx

```diff
@@ -79,80 +80,7 @@ export default function Index(): JSX.Element {
   return (
     <div className="flex flex-col gap-4">
       <div className="flex items-center justify-between">
-        <h1 className="text-[1.75rem] font-semibold ">My Projects</h1>
-
-        <Dialog>
-          <DialogTrigger>
-            <Button>
-              {' '}
-              <AddSVG /> Create a new Project
-            </Button>
-          </DialogTrigger>
-          <DialogContent>
-            <DialogHeader>Create a new project</DialogHeader>
-            <DialogDescription>
-              Fill in the details to create a new project
-            </DialogDescription>
-            <div className="flex flex-col gap-y-8">
-              <div className="flex w-full flex-col gap-y-4">
-                <div className="flex flex-col items-start gap-4">
-                  <Label className="text-right" htmlFor="name">
-                    Name
-                  </Label>
-                  <Input
-                    className="col-span-3"
-                    id="name"
-                    onChange={(e) => {
-                      setNewProjectData((prev) => ({
-                        ...prev,
-                        name: e.target.value
-                      }))
-                    }}
-                    placeholder="Enter the name"
-                  />
-                </div>
-                <div className="flex flex-col items-start gap-4">
-                  <Label className="text-right" htmlFor="name">
-                    Description
-                  </Label>
-                  <Input
-                    className="col-span-3"
-                    id="name"
-                    onChange={(e) => {
-                      setNewProjectData((prev) => ({
-                        ...prev,
-                        description: e.target.value
-                      }))
-                    }}
-                    placeholder="Enter the name"
-                  />
-                </div>
-                {/* {isNameEmpty ? (
-                  <span className="ml-[3.5rem] mt-1 text-red-500">
-                    Name cannot be empty
-                  </span>
-                ) : null} */}
-              </div>
-            </div>
-            <div className="flex w-full justify-end">
-              <Button
-                onClick={() => {
-                  Projects.createProject(newProjectData, currentWorkspace.id)
-                    .then(() => {
-                      toast.success('New project added successfully')
-                      router.refresh()
-                    })
-                    .catch(() => {
-                      toast.error('Failed to add new project')
-                    })
-                }}
-                variant="secondary"
-              >
-                Add project
-              </Button>
-            </div>
-          </DialogContent>
-        </Dialog>
+        {/* <h1 className="text-[1.75rem] font-semibold ">My Projects</h1> */}
```

Comment:
Not sure about this commented out line. I feel this should be removed.

Reply:
ah yes i forgot to remove it. I'll do it now